src-electron/db/query-command.js

```diff
@@ -2046,6 +2048,50 @@ async function selectNonManufacturerSpecificCommandDetailsFromAllEndpointTypesAn
   )
 }
 
+/**
+ * Get all commands in an endpoint type cluster
+ * Non-required commands are not loaded into ENDPOINT_TYPE_COMMAND table,
```

Comment:
Are non-required attributes loaded into endpoint_type_attribute table? I don't see deviceTypeClusterId being used in selectAttributesByEndpointTypeClusterId

Reply:
Yes, if you create an endpoint and check ENDPOINT_TYPE_ATTRIBUTE table, there will be attributes with INCLUDED = 0, so no deviceTypeClusterId is needed.